### ISSUE TITLE
Update issue with storage package import

### ISSUE THREAD

**Issue by tgalery** [semanticize/st#8]
Hi, I tried to update st using `go get -u github.com/semanticize/st/...` and I got the following error:

```
package github.com/semanticize/st/cmd/semanticizest
    imports github.com/alecthomas/units
    imports github.com/cheggaaa/pb
    imports github.com/mattn/go-sqlite3
    imports github.com/semanticize/st/hash
    imports github.com/semanticize/st/hash/countmin
    imports github.com/semanticize/st/internal/storage
    imports github.com/semanticize/st/wikidump
    imports golang.org/x/text/transform
    imports golang.org/x/text/unicode/norm
    imports github.com/semanticize/st/linking
    imports github.com/semanticize/st/nlp
    imports github.com/semanticize/st/storage
    imports github.com/semanticize/st/storage
    imports github.com/semanticize/st/storage: cannot find package "github.com/semanticize/st/storage" in any of:
    /usr/lib/go/src/pkg/github.com/semanticize/st/storage (from $GOROOT)
    /home/intruder/goworkspace/src/github.com/semanticize/st/storage (from $GOPATH)
```

**Comment by larsmans** [comment]
Does the problem go away if you issue the command a second time? Because if so, this may be golang/go/issues/9224.

**Comment by tgalery** [comment]
Right on, yeah it it did go away, but now I got the following:

```
# github.com/mattn/go-sqlite3
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
Go type not supported in export: [0]byte
```

**Comment by larsmans** [comment]
Appears to be mattn/go-sqlite3#245. Which Go version do you have?

**Comment by tgalery** [comment]
`go1.2.1 linux/amd64` . It looks we need to use go >= 1.4 . It might be worth putting that on the readme. If you want I can submit a PR.

**Comment by larsmans** [comment]
Please do, and remove 1.2 from `.travis.yml` then too. I wasn't aware that go-sqlite3 updated their dependencies; this explains the test failures I got.

**Comment by tgalery** [comment]
:+1:

**Comment by tgalery** [comment]
Solved with #9
